sdk/src/types/block/payload/tagged_data.rs

```diff
@@ -1,7 +1,8 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! Module describing the tagged data payload.
+//! A basic payload type that allows the addition of arbitrary data.
+//! [TIP-53: Tagged Data](https://github.com/iotaledger/tips/blob/tip53/tips/TIP-0053/tip-0053.md).
```

Comment:
I feel like this doc should be on the type itself, and I would keep the module doc as it was.

Reply:
I'd rather keep it like that tbh, because I'll do the same for them all so it's consistent. The content of a TIP may affect more than just the type so I think module wise make sense.

Reply:
I'm fine with it as-is

Reply:
okay